examples/kubernetes/namespace-template.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: {{namespace}}
+  labels:
+    name: {{namespace}}
```

Comment:
I don't think this label is useful. Also I was told once that using "name" as a label name is discouraged, because it gets confused with the metadata name field.